.github/workflows/pre-release.yml

```diff
@@ -8,7 +8,7 @@ on:
       - "v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+"      # e.g. v0.37.0-rc1, v0.38.0-rc10
 
 jobs:
-  goreleaser:
+  prerelease:
```

Comment:
Does this change imply we need to rename as well in lines 24 and 34 below?

Reply:
No, the change here just changes the job ID (like a variable name). The references to goreleaser on the other lines are important to keep as-is because they indicate the details of the GitHub action to execute.